sync/clients/sync.py

```diff
@@ -144,6 +144,20 @@ def get_project_recommendation(self, project_id: str, recommendation_id: str) ->
             )
         )
 
+    def get_project_submissions(self, project_id: str, params: dict = None) -> dict:
```

Comment:
`**params` might make for a friendlier interface so long as we don't want to call any params out.